WalletWasabi.Fluent/ViewModels/Wallets/Labels/LabelRanking.cs

```diff
@@ -0,0 +1,85 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Labels;
+
+public static class LabelRanking
```

Comment:
This shouldn't be in the `ViewModels` folder, this class seems more like a helper class. How about renaming it and moving it to the `Helpers` folder?